data_adapter_oemof/adapters.py

```diff
@@ -400,6 +402,59 @@ class VolatileAdapter(Adapter):
     facade = facades.Volatile
 
 
+class MIMOAdapter(Adapter):
+    """
+    MIMOAdapter
+    """
+
+    type = "mimo"
+    facade = MIMO
+    extra_fields = (
+        Field(name="name", type=str),
+        Field(name="region", type=str),
+        Field(name="year", type=int),
+        Field(name="groups", type=dict),
+        Field(name="capacity_cost", type=dict),
+    )
+
+    def get_default_parameters(self) -> dict:
+        defaults = super().get_default_parameters()
+        defaults["groups"] = self.get_groups()
+        keywords = ("emission_factor_", "conversion_factor_", "flow_share_")
```

Comment:
Conversion factors do work indeed. I will check what is preventing emission_factors them on occurring.

Reply:
Ah yes indeed, conversion factor is working but for emission factor there is a type, they are named "emissions_factor".
Where do I find specification for the correct form?

Reply:
this would not work - as facade expects keyword `emission_factor`.
Thus, we have to either remove the `s` within the FacadeAdapter or change it within the Facade.
I vote for the first one, as for me wording sounds better? and it is the easiest solution right now.
Moving forward...

Reply:
Sorry, my mind got confused and forgot that we have such a nice thing as a mapping for that :)